### PR TITLE
Adds documentation for fingerprint

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -218,6 +218,60 @@ Options:
...
+Adds a fingerprint to the project configuration.
+
+Usage:
+
+```
+bubblewrap fingerprint add [SHA-256 fingerprint] <flags>
+```
+
+Flags:
+ - `--name=<name>`: he name for the fingerprint.
```
Comment (PEConn) [inline packages/cli/README.md:246]: nit: s/he/the/

File: packages/cli/README.md [hunk 2]
```diff
@@ -218,6 +218,60 @@ Options:
...
+
+#### `remove`
+Removes a fingerprint from the project configuration.
+
+Usage:
+```
+bubblewrap fingerprint remove [SHA-256 fingerprint] <flags>
```
Comment (PEConn) [inline packages/cli/README.md:253]: For the above command you mention the flags below the example, but you don't do it here.

File: packages/cli/README.md (hunk 1 above)
Comment (PEConn) [inline packages/cli/README.md:246]: Could you add something saying what the `name` does?

File: packages/cli/README.md (hunk 2 above)
Comment (andreban) [reply packages/cli/README.md:253]: There are no flags beyond the global for this command.

Comment (PEConn) [reply packages/cli/README.md:253]: Ah, OK, in which case, can you rename "Global Options" to "Global Flags" above, and rename Flags on line 245 to "Additional Flags"?

File: packages/cli/README.md (hunk 1 above)
Comment (andreban) [reply packages/cli/README.md]: updated.

Comment (andreban) [reply packages/cli/README.md]: done.

**Comment by PEConn** [comment]
LGTM, thanks

**Pull request merged**